documentation/commands/ship.md

```diff
@@ -26,6 +26,9 @@ into the main branch, resulting in linear history on the main branch.
 Only shipping of direct children of the main branch is allowed.
 To ship a nested child branch, all ancestor branches have to be shipped or killed.
 
+If you have a pull request for your branch on GitHub and the environment variable `GIT_TOWN_GITHUB_TOKEN` set to a GitHub personal access token with access to your repositories,
```

Comment:
What about also allowing to store this configuration setting in the global Git config?
- store the setting: `git config --global git-town.github-token XXX`
- read the setting: `git config git-town.github-token`

Advantages:
- this doesn't require the user to modify environment variables. This is a significant setup step that non-technical users have to be guided through. It often requires determining the type of shell they use, figuring out how their particular shell configuration is set up (there are multiple files that can contain Bash config), potentially having them create new config files like `bash_profile` etc, and having the user restart the shell/computer (depending on which file is modified) 
- it is easier to use different tokens in different repos by storing them in the local Git repo